### ISSUE TITLE
T12 load out configurations

### ISSUE THREAD

**Issue by shockthealien** [DestinyItemManager/DIM#802]
Loadout Builder is great, but I've come across a small issue with actually using its results.

Differently from other types of loadouts you can build, the loadouts you can build from the Builder to achieve T12 have to be configured in a specific manner, but once the load out is found and built, the explanation on how to configure each item (int, dis, str) to actually achieve t12 is lost and the only way to find out is to bo back in the builder and build it again

So in short if the load out builder could keep record of how to configure each item to achieve T12 somehow it would be great

**Comment by Gr0m** [comment]
+1

I have items that are shared between multiple loadouts, but with different configuration (like it was INT for the first buildout and DIS for the second). And every time I switch my loadouts, I have to run T12 builder again to reconfigure my buildout correctly.

**Comment by kyleshay** [comment]
thanks for your feedback! we'll continue to track this in #720
